README.md

```diff
@@ -1,7 +1,19 @@
-# multinet
-- A Flask application serving the Multinet API
-- A VueJS client for managing, exploring, visualizing, and sharing your Multinet
-    data
+# multinet-server
 
-## Documentation
-https://multinet-app.readthedocs.io/en/latest/
+This is the server component for the Multinet project. It is a Flask application
+that serves the Multinet REST API.
+
+## Quick Start
+
+To get the server up and running in dev mode:
+
+1. Ensure that you have Python 3.7 and Pipenv installed on your system.
+2. Check out this repository and move into it (e.g., `cd multinet-server`).
+3. Install the Pipenv dependencies: `pipenv install`.
+4. Activate the virtual environment: `pipenv shell`.
+5. Run the server application: `pipenv run serve`.
+6. Visit http://localhost:5000 to ensure that the server is working.
```

Comment:
It's actually not required to activate the virtual environment here (pipenv doesn't install pipenv unless you manually add it), however I'm fine with leaving it if you feel strongly about this (i.e. just a place to show how to activate the virtual env).

```suggestion
4. Run the server application: `pipenv run serve`.
5. Visit http://localhost:5000 to ensure that the server is working.
```

Reply:
Nah, looks good to me.